docs/modelserving/v1beta1/llm/huggingface/text_generation/README.md

```diff
@@ -13,6 +13,7 @@ If the model is not supported by vLLM, KServe falls back to HuggingFace backend
 Create a secret with the Hugging Face token.
 
 === "Yaml"
```

Comment:
wouldn't be better to change the name from yaml to Secret and to Inference Service?

Reply:
We want them to be separate. Otherwise people would miss it

Reply:
Yes, I meant, there are two tabs called Yaml.
would be better to rename these tabs to secret and isvc.